### DISCUSSION TITLE
Started Building the MEGA ELS

### DISCUSSION THREAD

**Discussion by Lester444** [kachurovskiy/megaels#19] [Show and tell]
Hello Maxim.

I have started to build the MEGA ELS. I used 2GT timing belt & pulleys (3:1 ratio) and tested out its operation. Seems to work ok but I think it needs a bigger NEMA17 motor. I will get one soon.
https://youtu.be/wGypAkerZ-w

I am lost with the MEGA code. I see several ___.ino files in the github folder. How do I compile all of these files and load it into MEGA?

**Comment by kachurovskiy** [comment]
Hi @Lester444 - just download the entire repository to your computer, open `Digital_Feed_7e2/Digital_Feed_7e2.ino` in Arduino IDE and it should automatically detect all other filed in the folder.

More details in https://github.com/kachurovskiy/megaels#code - thanks!
